gpii/node_modules/deviceReporter/src/DeviceGet.js

```diff
@@ -22,18 +22,45 @@
         invokers: {
             handle: {
                 funcName: "gpii.handleDeviceGet",
-                args: ["{requestProxy}", "{deviceReporter}.installedSolutionsDataSource", "{deviceReporter}.platformReporter"]
+                args: [
+                    "{requestProxy}",
+                    "{deviceReporter}.installedSolutionsDataSource",
+                    "{deviceReporter}.platformReporter",
+                    "{deviceReporter}.solutionsRegistryDataSource"
+                ]
             }
         }
     });
 
-    gpii.handleDeviceGet = function (requestProxy, installedSolutionsDataSource, platformReporter) {
-        installedSolutionsDataSource.get(null, function onSuccess(solutions) {
-            requestProxy.events.onSuccess.fire({
-                solutions: solutions,
-                OS: platformReporter.reportPlatform()
+    gpii.handleDeviceGet = function (requestProxy, installedSolutionsDataSource, platformReporter, solutionsRegistryDataSource) {
```

Comment:
@amb26, splitted into 'static' and 'live' [here](https://github.com/javihernandez/universal/commit/c8d31dd8bf246dcd1a679b14d93e9a93b06e901f), but I still need to address the "Other notes".

ii) how can I use the flowManager's solutionsRegistryDataSource? By having replaced the current {solutionsRegistryDataSource} with {flowManager}.solutionsRegistryDataSource in my argument list didn't do the trick?
iii) How? Should I wrap this into a callback for deviceReporter.get? Or should I add a new invoker in gpii.deviceReporter.base?